Tests/PackageRegistryTests/RegistryManagerTests.swift

```diff
@@ -157,20 +164,20 @@ final class RegistryManagerTests: XCTestCase {
 
     // FIXME: this fails with error "the package manifest at '/Package.swift' cannot be accessed (/Package.swift doesn't exist in file system)"
```

Comment:
Can we uncomment the test?

Reply:
tried to fix it but this is still not working correctly. i plan to address in a separate pr to control scope of this one

Reply:
ok